### PR TITLE
Display tickets after purchase

### PR DESCRIPTION
**Pull request by kathrynowy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "redux-thunk": "^2.3.0",
     "reset-css": "^4.0.1",
     "socket.io-client": "~2.2.0",
+    "stripe": "^6.31.0",
```
Comment (dzmitry-penda) [inline package.json]: You dont need stripe

File: src/App.js [hunk 2]
```diff
@@ -66,6 +65,7 @@ class App extends Component {
         <Router history={history}>
           <div className="container">
             <PrimarySearchAppBar click={this.drawToggleClickHandler} />
+           {/*  <Ticket /> */}
```
Comment (dzmitry-penda) [inline src/App.js]: Remove this

File: src/App.scss [hunk 3]
```diff
@@ -16,12 +16,13 @@
...
+  background: linear-gradient(15deg,$light-black 60%,$dark-burgundy);
+   background: no-repeat url(./posters-bg.svg) center top/100% auto,linear-gradient(to bottom,$dark-burgundy,$dark-burgundy); 
   color: $white;
   display: flex;
   flex: 1 1 auto;
   flex-direction: column;
   padding-bottom: 30px;
   min-width: 350px;
-  color: $whiteForFont;
+  color: $white-for-font;
```
Comment (dzmitry-penda) [inline src/App.scss]: why not white? bad naming

File: src/Components/CheckoutForm/CheckForm.scss [hunk 4]
```diff
@@ -26,6 +26,7 @@ label {
     padding: 0 14px;
     box-shadow: 0 4px 6px $payment-form-shadow-blue, 0 1px 3px $payment-form-shadow-black;
     color: $white !important;
+    background-color: $bright-pink !important;
```
Comment (dzmitry-penda) [inline src/Components/CheckoutForm/CheckForm.scss]: This should not be !important

File: src/reducers/movies.js [hunk 5]
```diff
@@ -1,9 +1,10 @@
-import { GET_MOVIES_SUCCESS, GET_MOVIES_FAILURE, FILTER_MOVIES_SUCCESS } from '../constans/actionTypes.js'
+import { GET_MOVIE_SUCCESS, GET_MOVIES_SUCCESS, GET_MOVIES_FAILURE, FILTER_MOVIES_SUCCESS } from '../constans/actionTypes.js'
 
 const initialState = {
   allMovies: [],
   filteredMovies: [],
-  error: {}
+  error: {},
+  movieById: {}
```
Comment (dzmitry-penda) [inline src/reducers/movies.js]: just movie

File: src/Components/CheckoutForm/CheckForm.scss (hunk 4 above)
Comment (kathrynowy) [reply src/Components/CheckoutForm/CheckForm.scss]: it's material component

File: src/Components/Hall/ConfirmTicket/ConfirmTicket.js [hunk 6]
```diff
@@ -67,13 +67,13 @@ class ConfirmTicket extends Component {
     });
 
     this.props.buyTickets(confirmedTickets);
+    this.props.displayBoughtTickets(confirmedTickets, url);
   }
 
   async buyConfirmedTickets(token, url, totalCost) {
-    let { data } = await axios.post(`${apiBaseUrl}payment`, { token, totalCost });
+    let { data } = await axios.post("payment", { token, totalCost });
```
Comment (dzmitry-penda) [inline src/Components/Hall/ConfirmTicket/ConfirmTicket.js]: use single style of brackets, '' or ""

**Pull request merged**